packages/error-reporting/README.md

```diff
@@ -110,7 +115,7 @@ When running on Google Cloud Platform, we handle these for you automatically.
 
 ## Configuration
 
-The following code snippet lists all available configuration options. All configuration options are optional.
+The following code snippet lists all available configuration options. Except for the `keyFilename` and `credentials` options, all configuration options are optional.  Whether or not the `keyFilename` and `credentials` options are required is outlined in the [Running on Google Cloud Platform](#running-on-google-cloud-platform) section above.
```

Comment:
Aren't `keyFilename` and `credentials` optional too, specially on GCP? What am I missing?